init.lua

```diff
@@ -679,6 +679,10 @@ require('lazy').setup({
           -- Select the [p]revious item
           ['<C-p>'] = cmp.mapping.select_prev_item(),
 
+          -- scroll [u]p and [d]own the documentation window
```

Comment:
the comment seems to indicate that `u` and `d` keys are used but that's not the case.

Reply:
dangit. sorry, forgot to fix the comment